app.py

```diff
@@ -72,11 +73,20 @@ def frame():
     signer = Signer(SECRET_KEY)
     try:
         file_url = signer.unsign(request.args.get('file_url'))
-        return render_template('frame.html', file=file_url)
+        print str(file_url)
```

Comment:
For logs you should use python's logging library. Print should not be used in production as stdout can be redirected somewhere from where you can not read (i.e. `/dev/null`).